install.js

```diff
@@ -304,6 +305,14 @@ function requestBinary(requestOptions, filePath) {
   return deferred.promise
 }
 
+function getTargetPlatform(){
+  if (process.env.PHANTOMJS_PLATFORM){
```

Comment:
i poked around a bit, and i think the idiomatic way to support cross-compilation is with process.env.npm_config_platform and process.env.npm_config_arch? That way, this will integrate well with how npm does configuration in other places.